framework/src/vectorpostprocessors/LineValueSampler.C

```diff
@@ -65,3 +72,38 @@ LineValueSampler::generatePointsAndIDs(const Point & start_point,
   points[num_points - 1] = end_point;
   ids[num_points - 1] = (end_point - start_point).norm();
 }
+
+Real
+LineValueSampler::getValue(Point p) const
+{
+  if (_values.size() != 1)
+    mooseError("LineValueSampler: When using LineValueSampler with MultiAppUserObjectTransfer, "
```

Comment:
The `getValue()` method could be called by classes other than `MultiAppUserObjectTransfer`, so make these messages more general:

"When calling getValue() on LineValueSampler, only one variable can be provided as input to LineValueSampler"

Same deal with the error below.